src/main/java/autoracing/RacingGame.java

```diff
@@ -27,8 +31,30 @@ public void start() {
         }
     }
 
+    public int getTotalRounds() {
+        return totalRounds;
+    }
+
+    public List<Car> getParticipants() {
+        return participants;
+    }
+
     public void replayResult() {
-        ResultView resultView = new ResultView(participants, totalRounds, "-", "실행 결과");
+        ResultView resultView = new ResultView(this, "-", "실행 결과");
         resultView.show();
     }
+
+    public List<Car> getWinners() {
+        int winnerDistance = getWinnerDistance();
+        return participants.stream()
+                .filter((car) -> car.getLocation(totalRounds).getDistance() == winnerDistance)
+                .collect(Collectors.toList());
+    }
+
+    private int getWinnerDistance() {
+        Car oneOfWinner = participants.stream()
+                .max(Comparator.comparingInt(a -> a.getLocation(totalRounds).getDistance()))
+                .orElseThrow(() -> new IllegalArgumentException(""));
+        return oneOfWinner.getLocation(totalRounds).getDistance();
```

Comment:
우승자의 위치를 반환하는 거면 Stream 자체에서 반환해도 될 것 같은데 어떻게 생각하시나요??

```suggestion
        return participants.stream()
                .map(car -> car.getLocation(totalRounds).getDistance)
                .max()
                .orElseThrow(() -> new IllegalArgumentException(""));
```
(IDE에서 작성한 코드가 아니라서 오류가 있을 수 있습니다:) 참고만 해주세요)